doc/Extensions/Applications.md

```diff
@@ -2435,6 +2435,92 @@ Also if the system you are using uses non-static device naming based
 on bus information, it may be worthwhile just using the SN as the
 device ID is going to be irrelevant in that case.
 
+## Sneck
+
+If you want to replace Nagios/Icinga, especially in regards to NRPE,
+this works great. This allows LibreNMS to query what checks were ran
+on the server and keep track of totals of OK, WARNING, CRITICAL, and
+UNKNOWN statuses. The big advantage over this compared to a NRPE with
+the Services extension is you don't need to know what checks are
+configured on it. Also don't need to wait for the tests to run as
+sneck is meant to be ran via cron and the then return the cache when
+queried via SNMP, meaning a lot faster response time, especially if
+slow checks are being performed.
+
+Included are alert examples. Although if you want to set up your own,
+the metrics below are provided.
+
+| Metric | Description |
+|--------|------------|
+| ok     | Total OK checks |
+| warning | Total WARNING checks |
+| critical | Total CRITICAL checks |
+| unknown | Total UNKNOWN checks |
+| errored | Total checks that errored |
+| time_to_polling | Differnce in seconds between when polling data was generated and when polled |
+| time_to_polling_abs | The aboslute value of time_to_polling. |
+
+You can check the exit status of a specific check via the metric
+`check_$CHECK` where $CHECK is equal to the name of the check in
+questions. So if the check is named `foo` you can check the exit
+status by checking `check_foo`. The standard Nagios/Icinga style exit
+codes are used and those are as below.
+
+| Exit | Meaning  |
+|------|----------|
+| 0    | okay     |
+| 1    | warning  |
+| 2    | critical |
+| 3+   | unknown  |
+
+If you wish to use `time_to_polling`, you will need to set in the
+config the item below. The default is false. Unless set to true, this
+value will default to 0. If you plan to enable this, you will want
+to make sure you are using NTP every were or it will alert if it goes
+over a difference of 540s.
+
+```
+$config['apps']['sneck']['polling_time_diff'=true;
```

Comment:
use `lnms config:set` commands instead

Reply:
@Jellyfrog awesome, done! :)

Also cleaned up the docs and added a config definition for it. :)